erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py

```diff
@@ -145,65 +164,159 @@ def validate_item(self, item_code, row_num):
 			validate_is_stock_item(item_code, item.is_stock_item, verbose=0)
 
 			# item should not be serialized
-			if item.has_serial_no == 1:
-				raise frappe.ValidationError(_("Serialized Item {0} cannot be updated using Stock Reconciliation, please use Stock Entry").format(item_code))
+			if item.has_serial_no and not row.serial_no and not item.serial_no_series:
+				raise frappe.ValidationError(_("Serial nos are required for serialized item {0}").format(item_code))
```

Comment:
`_("Serial no(s) required for serialized item {0}"`

Reply:
Done